sdk/metric/benchmark_test.go

```diff
@@ -369,3 +372,89 @@ func benchCollectAttrs(setup func(attribute.Set) Reader) func(*testing.B) {
 		b.Run("Attributes/10", run(setup(attribute.NewSet(attrs...))))
 	}
 }
+
+func BenchmarkExemplars(b *testing.B) {
+	sc := trace.NewSpanContext(trace.SpanContextConfig{
+		SpanID:     trace.SpanID{0o1},
+		TraceID:    trace.TraceID{0o1},
```

Comment:
nit: Any reason why `0o1`? Maybe `0x1`?

Reply:
They need to be non-zero. Having octals makes no difference from hex.

Reply:
I know but octal looks strange

Reply:
I feel as the author of these changes I am extended some autonomy in making the choice of what nonce value out the 255 numbers a span ID can be and 65531 a trace ID can be that I want to include in the change.

If octal numbers look strange to you, I'm not sure what to say. They are a part of the Go specification and valid number representations.

You have unresolved this comment. Are you blocking this PR on this?

Reply:
No I am not blocking. Resolving.